package.json

```diff
@@ -41,7 +41,7 @@
     "@reason-native-web/h1": ">= 1.0.0-alpha.1 < 2.0.0",
     "@reason-native-web/h1-lwt-unix": ">= 1.0.0-alpha.1 < 2.0.0",
     "@reason-native-web/piaf": ">= 1.0.0-alpha.1 < 2.0.0",
-    "@reason-native-web/ssl": ">= 0.5.9-dev.1 < 1.0.0"
+    "@reason-native-web/ssl": "0.5.9002"
```

Comment:
Can we use a version range here? I'm thinking `">= 0.5.9002 < 1.0.0"`

Reply:
Yes I guess it should be ok. I just copied what morph uses (fixed version)

Reply:
Great! Merging.